.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -375,3 +448,13 @@ jobs:
         grep 'exit code: [^0]' -r logs >> $GITHUB_STEP_SUMMARY
 
         exit $ret
+    - name: Pack e2e binaries
+      if: ${{ always() && !cancelled() && inputs.upload_artifact == 'true'}}
+      run: tar -I '${{ steps.artifact_info.outputs.COMPRESS }}' -cf ${{ steps.artifact_info.outputs.ARCHIVE_NAME }} -C ./build-e2e .
+    - name: Upload e2e binaries
+      if: ${{ always() && !cancelled() && inputs.upload_artifact == 'true'}}
+      uses: actions/upload-artifact@v4
+      with:
+        name: sycl_e2e_bin_${{ inputs.build_artifact_suffix }}
+        path: ${{ steps.artifact_info.outputs.ARCHIVE_NAME }}
+        retention-days: ${{ inputs.retention-days }}
```

Comment:
i dont see `retention-days` as input, so i dont understand how this can not error out

Reply:
Not sure either, I'll add it and set it to 1